paasta_tools/mesos_tools.py

```diff
@@ -47,6 +49,7 @@
 
 
 DEFAULT_MESOS_CLI_CONFIG_LOCATION = "/nail/etc/mesos-cli.json"
+DEFAULT_SERVICES_LOCATION = "/nail/etc/services/services.yaml"
```

Comment:
Can we make this configurable? If someone wants to use it outside of yelp, then they should be able to configure it rather than changing the code.